### DISCUSSION TITLE
Estimator not recognizing rv

### DISCUSSION THREAD

**Discussion by nbungart1999** [phoebe-project/phoebe2#459] [Questions]
So I am trying to add the rv_geometry solver to my system but it is warning me that 'rv' is not a recognized kwarg when I copied this code straight from the inverse problem examples page. This is also what is probably causing the run_solver part to generate an error as well because it says no solvers are attached. Any thoughts on how to fix this?
![image](https://user-images.githubusercontent.com/83960306/121227545-af158600-c859-11eb-8347-c34203f979f5.png)

**Comment by nbungart1999** [comment]
We did not find a solution for this particular method but in the actual rv estimators tutorial it has an updated version of how to do this so maybe the inverse problem examples page needs updated

**Reply by kecnry** [reply]
just pushed an update to the tutorial.  It should be `rv_datasets='rv01'`, not `rv='rv01'`.  Thanks!

**Reply by kecnry** [reply]
the other possible issue you may be running into is that your script should say `kind='rvgeometry'` (no '01').  Alternatively you can "tag" the solver options by passing `solver='myrvgeom'` (or whatever you'd like to call it) to the `add_solver` line and then pass that same name `solver='myrvgeom'` to the `run_solver` line in place of `kind='rvgeometry'`.

**Reply by nbungart1999** [reply]
The problem I had with kind='rvgeometry' is that once it ran it said it needed the 01, 02 and so on
